build.xml

```diff
@@ -165,13 +167,20 @@
         </condition>
         <echo if:set="saxon-available" message="saxon available: ${saxon-available}"/>
         <echo unless:set="saxon-available" message="saxon available: false"/>
+        <!-- check if Stylesheets are available -->
+        <condition property="stylesheets-available">
+            <available file="${dir.lib.stylesheets}"/>
```

Comment:
Tested this PR locally and with Docker, everything runs fine. Great, many thanks @rettinghaus !

However, I am wondering if this check is sufficient here? At the moment, it only checks if there is a folder named stylesheets. It does not care about the content. So you could have either an empty stylesheets folder or one with an older or newer version of the stylesheets or even one with a totally different content, and the check would return true. 

So what could be the best way to check for the correct content?

PS: I see that there is a VERSION file, and we definitely need the `/odd/build-to.xml`, `/html5/build-to.xml`, `/rng/build-to.xml` files. These call the `/common/teianttasks.xml` which references various other directories and files.

Reply:
This also applies when it comes to version updates. When you have 7.56 in your lib folder, but the build.xml would have been updated to 7.57, the build script would not download the newer version.

Reply:
I could argue that this only was a _proof of concept_. 😉 But you're absolutely right. Thanks for pointing this out!

Reply:
I changed the check to check the `VERSION` file for the expected version. From my point of view should that be sufficient: if anything doesn't work you can simply run `ant reset`.

Reply:
See comments below